### PR TITLE
update screenfetch recipe

### PR DESCRIPTION
**Pull request by Ra-V2**

### CODE REVIEW

File: app-misc/screenfetch/screenfetch-3.9.1.recipe [hunk 1]
```diff
@@ -10,7 +10,7 @@ COPYRIGHT="2014-2017 Brett Bohnenkamper"
 LICENSE="GNU GPL v3"
 REVISION="2"
```
Comment (return) [inline app-misc/screenfetch/screenfetch-3.9.1.recipe]: On every version update, REVISION must be changed to '1'.

```suggestion
REVISION="1"
```

**Comment by return** [comment]
Could you also reformat lines [41](https://github.com/haikuports/haikuports/pull/4426/files#diff-9720973cb0790f19861a3fad27162980R41) and [42](https://github.com/haikuports/haikuports/pull/4426/files#diff-9720973cb0790f19861a3fad27162980R42) to use a single tab instead of spaces please?

**Comment by aarroyoc** [comment]
Comes from: https://codein.withgoogle.com/dashboard/task-instances/5531622208700416/

**Comment by humdingerb** [comment]
Since aou all say it's OK, I merged. Approving GCI task now. Thanks!

**Pull request merged**